modules/mono/glue/GodotSharp/GodotSharp/Core/Callable.cs

```diff
@@ -131,12 +129,10 @@ public unsafe void CallDeferred(params Variant[] args)
                 stackalloc godot_variant.movable[VarArgsSpanThreshold].Cleared() :
                 new godot_variant.movable[argc];
 
-            Span<IntPtr> argsSpan = argc <= 10 ?
-                stackalloc IntPtr[argc] :
+            Span<IntPtr> argsSpan = argc <= VarArgsSpanThreshold ?
+                stackalloc IntPtr[VarArgsSpanThreshold] :
```

Comment:
The `VarArgsSpanThreshold` constant has a value of `5` in `Callable`, but `10` in code generated by bindings_generator. Not sure why they are different and maybe we should consider declaring this constant in a single place and reuse it rather than redefine it in every file.

Reply:
It's weird that I added `VarArgsSpanThreshold` here but used a hard-coded value instead. It's obviously a mistake, but I don't know which one I truly wanted. I think `10` should be fine for both.

Reply:
I'll change it to `10` in this PR and we can move the constant to a single place later.